go-controller/pkg/ovndbmanager/ovndbmanager.go

```diff
@@ -171,11 +182,20 @@ func ensureClusterRaftMembership(db string) {
 		}
 		memberFound := false
 		for _, knownServer := range knownServers {
-			if knownServer == matchedServer[1] {
+			if matchedServer[1] == knownServer {
 				memberFound = true
 				break
 			}
 		}
+		if !memberFound {
+			for _, masterPod := range masterPods.Items {
+				// should we look at PodIPs[0].IP instead?
```

Comment:
I *think* this is OK because even with dual stack we will only use the first IP (ipv4) I assume as the IPs for the db connections. @danwinship can confirm.

Reply:
Ok, will make the change.

Reply:
ah I had assumed ipv4 would always be first...of course nothing could be that simple